# Introduction

Molecular biology is the branch of biology that studies the molecular basis of biological activity. Living things are made of chemicals just as non-living things are, so a molecular biologist studies how molecules interact with one another in living organisms to perform the functions of life.

# DNA
DNA, or deoxyribonucleic acid, is the hereditary material in humans and almost all other organisms. Nearly every cell in a person’s body has the same DNA. Most DNA is located in the cell nucleus (where it is called nuclear DNA), but a small amount of DNA can also be found in the mitochondria (where it is called mitochondrial DNA or mtDNA).

# DNA Structure
The primary structure of DNA consists of a string of nucleotides joined together by phosphodiester linkages.

## Nucleotides

DNA is typically a very long molecule and is therefore termed a macromolecule. For example, within each human chromosome is a single DNA molecule that, if stretched out straight, would be several centimeters in length, thousands of times longer than the cell itself. In spite of its large size, DNA has a quite simple structure: it is a polymer – that is, a chain made up of many repeating units linked together. The repeating units of DNA are nucleotides, each comprising three parts:
1. a sugar
2. a nitogen-containing base
3. a phosphate

### 1. Sugars
The sugars of nucleic acids – called pentose sugars – have five carbon atoms, numbered 1′, 2′, 3′, and so forth. The sugars of DNA and RNA are slightly different in structure. RNA’s sugar, called ribose, has a hydroxyl group (–OH) attached to the 2′-carbon atom, whereas DNA’s sugar, or deoxyribose, has a hydrogen atom (–H) at this position and therefore contains one oxygen atom fewer overall. This difference gives rise to the names ribonucleic acid (RNA) and deoxyribonucleic acid (DNA). This minor chemical difference is recognized by all the cellular enzymes that interact with DNA or RNA, thus yielding specific functions for each nucleic acid. Furthermore, the additional oxygen atom in the RNA nucleotide makes it more reactive and less chemically stable than DNA. For this reason, DNA is better suited to serve as the long-term repository of genetic information.

These differences are illustrated by the images below:

![image](https://user-images.githubusercontent.com/98819046/153342314-dab5caee-f37f-42fe-9c5c-fd240f6fb193.png)

**A nucleotide contains either a ribose sugar (in RNA) or a deoxyribose sugar (in DNA).**  *The carbon atoms are assigned primed numbers*

### 2. Nitrogenous base
Nitrogenous bases may be of two types: a purine or a pyrimidine. Each purine consists of a six-sided ring attached to a five-sided ring, whereas each pyrimidine consists of a six-sided ring only. Both DNA and RNA contain two purines, adenine and guanine (A and G), which differ in the positions of their double bonds and in the groups attached to the six-sided ring. Three pyrimidines are common in nucleic acids: cytosine (C), thymine (T), and uracil (U). Cytosine is present in both DNA and RNA; however, thymine is restricted to DNA, and uracil is found only in RNA. The three pyrimidines differ in the groups or atoms attached to the carbon atoms of the ring and in the number of double bonds in the ring. In a nucleotide, the nitrogenous base always forms a covalent bond with the 1’-carbon atom of the sugar. A deoxyribose or a ribose sugar and a base together are referred to as a nucleoside.

![image](https://user-images.githubusercontent.com/98819046/153342937-ffb262d5-199d-4cb2-9fd7-33c8d060ed85.png)

**A nucleotide contains either a purine or a pyrimidine base.** *The atoms of the rings in the bases are assigned unprimed numbers.*

### 3. Phosphate group
The phosphate group, which consists of a phosphorus atom bonded to four oxygen atoms. Phosphate groups are found in every nucleotide and frequently carry a negative charge, which makes DNA acidic. The phosphate group is always bonded to the 5′-carbon atom of the sugar in a nucleotide.

The DNA nucleotides are properly known as deoxyribonucleotides or deoxyribonucleoside 5’-monophosphates. Because there are four types of bases, there are four different kinds of DNA nucleotides. The equivalent RNA nucleotides are termed ribonucleotides or ribonucleoside 5’-monophosphates. RNA molecules sometimes contain additional rare bases, which are modified forms of the four common bases.

![image](https://user-images.githubusercontent.com/98819046/153343364-b6e0b608-f0a0-455f-8237-3736807770b7.png)

The secondary structure of DNA refers to its three-dimensional configuration—its fundamental helical structure. DNA’s secondary structure can assume a variety of configurations, depending on its base sequence and the conditions in which it is placed.

### The double helix: 
a fundamental characteristic of DNA’s secondary structure is that it consists of two polynucleotide strands wound around each other—it’s a double helix. The sugar–phosphate linkages are on the outside of the helix, and the bases are stacked in the interior of the molecule. The two polynucleotide strands run in opposite directions—they are antiparallel, which means that the 5′ end of one strand is opposite the 3′ end of the other strand.

![image](https://user-images.githubusercontent.com/98819046/153343740-29df8dde-be97-4a7a-b09f-3cfd3837e957.png)

The strands are held together by two types of molecular forces. Hydrogen bonds link the bases on opposite strands. These bonds are relatively weak compared with the covalent phosphodiester bonds that connect the sugar and phosphate groups of adjoining nucleotides on the same strand.

The nature of the hydrogen bond imposes a limitation on the types of bases that can pair. Adenine normally pairs only with thymine through two hydrogen bonds, and cytosine normally pairs only with guanine through three hydrogen bonds. Because three hydrogen bonds form between C and G and only two hydrogen bonds form between A and T, C–G pairing is stronger than A–T pairing. The specificity of the base pairing means that, wherever there is an A on one strand, there must be a T in the corresponding position on the other strand, and, wherever there is a G on one strand, a C must be on the other. The two polynucleotide strands of a DNA molecule are therefore not identical but are complementary DNA strands. The complementary nature of the two nucleotide strands provides for efficient and accurate DNA replication.

The second force that holds the two DNA strands together is the interaction between the stacked base pairs. These stacking interactions contribute to the stability of the DNA molecule but do not require that any particular base follow another. Thus, the base sequence of the DNA molecule is free to vary, allowing DNA to carry genetic information.

## Amino acids

Amino acids are molecules that combine to form proteins. Amino acids and proteins are the building blocks of life.

When proteins are digested or broken down, amino acids are left. The human body uses amino acids to make proteins to help the body:
- Break down food
- Grow
- Repair body tissue
- Perform any other body functions
- As a source of energy by the body

Amino acids are classified into three groups:

- Essential amino acids
- Nonessential amino acids
- Conditional amino acids

### Essential amino acids:
Essential amino acids cannot be made by the body. As a result, they must come from food.
The 9 essential amino acids are: histidine, isoleucine, leucine, lysine, methionine, phenylalanine, threonine, tryptophan, and valine.

### Nonessential amino acids:
Nonessential means that our bodies can produce the amino acid, even if we do not get it from the food we eat. Nonessential amino acids include: alanine, arginine, asparagine, aspartic acid, cysteine, glutamic acid, glutamine, glycine, proline, serine, and tyrosine.

### Conditional amino acids:
Conditional amino acids are usually not essential, except in times of illness and stress.
Conditional amino acids include: arginine, cysteine, glutamine, tyrosine, glycine, ornithine, proline, and serine.
You do not need to eat essential and nonessential amino acids at every meal, but getting a balance of them over the whole day is important. A diet based on a single plant item will not be adequate, but we no longer worry about pairing proteins (such as beans with rice) at a single meal. Instead we look at the adequacy of the diet overall throughout the day.

| Amino Acid          | 3 - Letter code   | 1 - Letter code |
| :----               |       :----:      |            ---: |
| alanine             | ala               | A               |
| arginine            | arg               | R               |
| asparagine          | asn               | N               |
| aspartic acid       | asp               | D               |
| asparagine/asp acid | asx               | B               |
| cysteine            | cys               | C               |
| glutamic acid       | glu               | E               |
| glutamine           | gln               | Q               |
| glutamine/gln acid  | glx               | Z               |
| glycine             | gly               | G               |
| histidine           | his               | H               |
| Isoleucine          | ile               | I               |
| leucine             | leu               | L               |
| lysine              | lys               | K               |
| methionine          | met               | M               |
| phenylalanine       | phe               | F               |
| proline             | pro               | P               |
| serine              | ser               | S               |
| threonine           | thr               | T               |
| tryptophan          | trp               | W               |
| tyrosine            | tyr               | Y               |
| valine              | val               | V               |

# Flow of genetic information/Central dogma

The central dogma of molecular biology is an explanation of the flow of genetic information within a biological system, first stated by Francis Crick in 1956. He stated, “The central dogma of molecular biology deals with the detailed residue-by-residue transfer of sequential information. It states that such information cannot be transferred back from protein to either protein or nucleic acid”.

Within each cell the genetic information flows from DNA to RNA to protein. This flow of information is unidirectional and irreversible. The information carried within the DNA dictates the end product (protein) that will be synthesized. This information is the genetic code. Conversion of DNA encoded information to RNA is called transcription. The information from a mRNA is then translated to an amino acid sequence in the corresponding protein.

This can be illustrated as below:

![image](https://user-images.githubusercontent.com/98819046/153387750-77506a62-1b65-4dbf-a55f-389df4b70346.png)

### Replication

Before a cell can divide, it must duplicate all its DNA. The copying from DNA to DNA arguably is the fundamental step in the central dogma. A complex group of proteins called the replisome performs the replication of the information from the parent strand to the complementary daughter strand.

The replisome comprises:
- a helicase that unwinds the superhelix as well as the double-stranded DNA helix to create a replication fork
- SSB protein that binds open the double-stranded DNA to prevent it from reassociating
- RNA primase that adds a complementary RNA primer to each template strand as a starting point for replication
- DNA polymerase III that reads the existing template chain from its 3' end to its 5' end and adds new complementary nucleotides from the 5' end to the 3' end of the daughter chain
- DNA polymerase I that removes the RNA primers and replaces them with DNA.
- DNA ligase that joins the two Okazaki fragments with phosphodiester bonds to produce a continuous chain.

This process typically takes place during S phase of the cell cycle. It can be well described in the below diagram

![image](https://user-images.githubusercontent.com/98819046/153389007-24efe2ab-2908-4a95-95e5-9c52c3771aed.png)

### Transcription
Transcription is the first step of gene expression, in which a particular segment of DNA is copied into RNA (mRNA) by the enzyme RNA polymerase. It is the process by which the information in DNA is copied into messenger RNA (mRNA) for protein production.

![image](https://user-images.githubusercontent.com/98819046/153389438-b7303d4f-eb95-45af-8826-2e080de36157.png)

**Transcription of two genes** *(a) RNA polymerase moves from the 3′ end of the template strand, creating an RNA strand that grows in a 5′→3′ direction (because it must be antiparallel to the template strand). Note that some genes are transcribed from one strand of the DNA double helix; other genes use the other strand as the template. (b) A uracil is being added to the 3′ end of the transcript for gene* Growth is thus 5′→3′.

Briefly, transcription relies on the complementary pairing of bases. The two strands of the double helix separate locally, and one of the separated strands acts as a template. Next, free nucleotides are aligned on the DNA template by their complementary bases in the template. The free ribonucleotide A aligns with T in the DNA, G with C, C with G, and U with A. The process is catalyzed by the enzyme RNA polymerase, which attaches and moves along the DNA adding ribonucleotides in the growing RNA. Hence, already we see the two principles of base complementarity and binding proteins (in this case, the RNA polymerase) in action.

RNA growth is always in the 5′→3′ direction: in other words, nucleotides are always added at a 3′ growing tip. Because of the antiparallel nature of the nucleotide pairing, the fact that RNA is synthesized 5′→3′ means that the template strand must be oriented 3′→5′.


Transcription has three distinct stages: **initiation, elongation, and termination.**

#### Initiation

The regions of the DNA that signal initiation of transcription in prokaryotes are termed promoters. In the E. coli promoters appear in the form of two regions of partial homology appear in the genome. These regions have been termed the −35 and −10 regions because of their locations relative to the transcription initiation point.

The dissociative subunit of RNA polymerase, the σ factor, allows RNA polymerase to recognize and bind specifically to promoter regions. First, the holoenzyme searches for a promoter and initially binds loosely to it, recognizing the −35 and −10 regions. The resulting structure is termed a closed promoter complex. Then, the enzyme binds more tightly, unwinding bases near the −10 region. When the bound polymerase causes this local denaturation of the DNA duplex, it is said to form an open promoter complex. This initiation step, the formation of an open complex, requires the sigma factor.

![image](https://user-images.githubusercontent.com/98819046/153390368-ab8435d5-257b-4f29-9278-9926bf23d8b2.png)

**Initiation of transcription** *(a) RNA polymerase searches for a promoter site. (b) It recognizes a promoter site and binds tightly, forming a closed complex. (c) The holoenzyme unwinds a short stretch of DNA, forming an open complex. Transcription begins, and the σ factor is released*

#### Elongation
Shortly after initiating transcription, the sigma factor dissociates from the RNA polymerase. The RNA is always synthesized in the 5′→3′ direction, with nucleoside triphosphates (NTPs) acting as substrates for the enzyme.

The sequential addition of nucleotides takes place one at a time in the 5′-to-3′ direction. The chain grows by the formation of a bond between the 3′ hydroxyl end of the growing strand and a nucleoside triphosphate, releasing one pyrophosphate ion (PPi). This results in the net addition of one phosphate, which is incorporated into the backbone of the new strand. DNA grows by reaction with deoxyribonucleoside triphosphates, and RNA grows by reaction with ribonucleoside triphosphates. Certain sequences may cause stalling or pausing, which becomes critical for termination of transcription.

![image](https://user-images.githubusercontent.com/98819046/153390920-368d6345-ddc9-4118-89a6-0e01eea9a370.png)

**Transcription by RNA polymerase.** *An RNA strand is synthesized in the 5′→3′ direction from a locally single stranded region of DNA.*

#### Termination
RNA polymerase also recognizes signals for chain termination, which includes the release of the nascent RNA and the enzyme from the template. Bacteria use two different strategies for transcription termination: Rho-independent termination and Rho-dependent termination.

In Rho-independent transcription termination, also called intrinsic termination, RNA transcription stops when the newly synthesized RNA molecule forms a G-C-rich hairpin loop followed by a run of Us. When the hairpin forms, the mechanical stress breaks the weak rU-dA bonds, now filling the DNA-RNA hybrid. This pulls the poly-U transcript out of the active site of the RNA polymerase, in effect, terminating transcription.

In the "Rho-dependent" type of termination, a protein factor called "Rho" destabilizes the interaction between the template and the mRNA, thus releasing the newly synthesized mRNA from the elongation complex.

### Translation
In translation, messenger RNA (mRNA)—produced by transcription from DNA—is decoded by a ribosome to produce a specific amino acid chain, or polypeptide. The polypeptide later folds into an active protein and performs its functions in the cell. The ribosome facilitates decoding by inducing the binding of complementary tRNA anticodon sequences to mRNA codons. The tRNAs carry specific amino acids that are chained together into a polypeptide as the mRNA passes through and is "read" by the ribosome. The entire process is a part of gene expression.
In brief, translation proceeds in four phases:
1.	Initiation: The ribosome assembles around the target mRNA. The first tRNA is attached at the start codon.
2.	Elongation: The tRNA transfers an amino acid to the tRNA corresponding to the next codon.
3.	Translocation: The ribosome then moves (translocates) to the next mRNA codon to continue the process, creating an amino acid chain.
4.	Termination: When a stop codon is reached, the ribosome releases the polypeptide.

In bacteria, translation occurs in the cell's cytoplasm, where the large and small subunits of the ribosome bind to the mRNA. In eukaryotes, translation occurs in the cytosol or across the membrane of the endoplasmic reticulum in a process called vectorial synthesis. In many instances, the entire ribosome/mRNA complex binds to the outer membrane of the rough endoplasmic reticulum (ER); the newly created polypeptide is stored inside the ER for later vesicle transport and secretion outside of the cell.

Many of transcribed RNA, such as transfer RNA, ribosomal RNA, and small nuclear RNA, do not undergo translation into proteins.

Transcription termination in eukaryotes is less understood but involves cleavage of the new transcript followed by template-independent addition of adenines at its new 3' end, in a process called polyadenylation. 

# Mutations

DNA is a highly stable molecule that is replicated with amazing accuracy, but changes in DNA structure and errors of replication do occur. A mutation is defined as an inherited change in genetic information; the descendants may be cells or organisms.

## Importance of mutations

Mutations are both the sustainer of life and the cause of great suffering. On the one hand, mutation is the source of all genetic variation, the raw material of evolution. The ability of organisms to adapt to environmental change depends critically on the presence of genetic variation in natural populations, and genetic variation is produced—at least partly—by mutation. On the other hand, many mutations have detrimental effects, and mutation is the source of many diseases and disorders.

Much of the study of genetics focuses on how variants produced by mutation are inherited; genetic crosses are meaningless if all individual members of a species are identically homozygous for the same alleles. Much of Gregor Mendel’s success in unraveling the principles of inheritance can be traced to his use of carefully selected variants of the garden pea; similarly, Thomas Hunt Morgan and his students discovered many basic principles of genetics by analyzing mutant fruit flies.

Mutations are also useful for probing fundamental biological processes. Finding or creating mutations that affect different components of a biological system and studying their effects can often lead to an understanding of the system. This method, referred to as genetic dissection, is analogous to figuring out how an automobile works by breaking different parts of a car and observing the effects; for example, smash the radiator and the engine overheats, revealing that the radiator cools the engine. Using mutations to disrupt function can likewise be a source of insight into biological processes. For example, geneticists have begun to unravel the molecular details of development by studying mutations, such as tinman, that interrupt various embryonic stages in Drosophila. Although this method of breaking “parts” to determine their function might seem like a crude approach to understanding a system, it is actually very powerful and has been used extensively in biochemistry, developmental biology, physiology, and behavioral science (but this method is not recommended for learning how your car works!).

## Categories of Mutations

In multicellular organisms, we can distinguish between two broad categories of mutations: somatic mutations and germ-line mutations. Somatic mutations arise in somatic tissues, which do not produce gametes. When a somatic cell with a mutation divides (mitosis), the mutation is passed on to the daughter cells, leading to a population of genetically identical cells (a clone). The earlier in development that a somatic mutation occurs, the larger the clone of cells within that individual organism that will contain the mutation.

Because of the huge number of cells present in a typical eukaryotic organism, somatic mutations are numerous. For example, there are about 1014 cells in the human body. Typically, a mutation arises once in every million cell divisions, and so hundreds of millions of somatic mutations must arise in each person. Many somatic mutations have no obvious effect on the phenotype of the organism, because the function of the mutant cell (even the cell itself) is replaced by that of normal cells. However, cells with a somatic mutation that stimulates cell division can increase in number and spread; this type of mutation can give rise to cells with a selective advantage and is the basis for cancers.

Germ-line mutations arise in cells that ultimately produce gametes. A germ-line mutation can be passed to future generations, producing individual organisms that carry the mutation in all their somatic and germ-line cells. When we speak of mutations in multicellular organisms, we’re usually talking about germ-line mutations.

## Types of Gene Mutations

Mutations are classified by their molecular nature as follows:

### Base substitutions
The simplest type of gene mutation is a base substitution, the alteration of a single nucleotide in the DNA. Base substitutions are of two types: In a transition, a purine is replaced by a different purine or, alternatively, a pyrimidine is replaced by a different pyrimidine. In a transversion, a purine is replaced by a pyrimidine or a pyrimidine is replaced by a purine.

### Insertions and deletions

The second major class of gene mutations contains insertions and deletions—the addition or the removal, respectively, of one or more nucleotide pairs (collectively called indels). Although base substitutions are often assumed to be the most common type of mutation, molecular analysis has revealed that insertions and deletions are often more frequent. Insertions and deletions within sequences that encode proteins may lead to frameshift mutations, changes in the reading frame of the gene. Frameshift mutations usually alter all amino acids encoded by nucleotides following the mutation, and so they generally have drastic effects on the phenotype. Not all insertions and deletions lead to frameshifts, however; insertions and deletions consisting of any multiple of three nucleotides will leave the reading frame intact, although the addition or removal of one or more amino acids may still affect the phenotype. Mutations not affecting the reading frame are called in-frame insertions and deletions, respectively.

### Mutations by other classifications

Expanding nucleotide repeats: Mutations in which the number of copies of a set of nucleotides increases in number are called expanding nucleotide repeats. This type of mutation was first observed in 1991 in a gene called FMR-1, which causes fragile-X syndrome, the most common hereditary cause of mental retardation. The mechanism that leads to the expansion of nucleotide repeats is not completely understood.

A mutation that alters the wild-type phenotype is called a forward mutation, whereas a reverse mutation (a reversion) changes a mutant phenotype back into the wild type.

A **base substitution** that results in a different amino acid in the protein is referred to as a missense mutation. A nonsense mutation changes a sense codon (one that specifies an amino acid) into a nonsense codon (one that terminates translation). If a nonsense mutation occurs early in the mRNA sequence, the protein will be greatly shortened and will usually be nonfunctional.

A **silent mutation** changes a codon to a synonymous codon that specifies the same amino acid, altering the DNA sequence without changing the amino acid sequence of the protein.

A **neutral mutation is a missense mutation** that alters the amino acid sequence of the protein but does not change its function. Neutral mutations occur when one amino acid is replaced by another that is chemically similar or when the affected amino acid has little influence on protein function.

**Loss-of-function mutations** cause the complete or partial absence of normal protein function. A loss-of-function mutation so alters the structure of the protein that the protein no longer works correctly or the mutation can occur in regulatory regions that affect the transcription, translation, or splicing of the protein.

In contrast, a gain-of-function mutation produces an entirely new trait or it causes a trait to appear in an inappropriate tissue or at an inappropriate time in development.
A suppressor mutation is a genetic change that hides or suppresses the effect of another mutation. This type of mutation is distinct from a reverse mutation, in which the mutated site changes back into the original wild-type sequence.

An **intragenic suppressor mutation** is in the same gene as that containing the mutation being suppressed and may work in several ways. The suppressor may change a second nucleotide in the same codon altered by the original mutation, producing a codon that specifies the same amino acid as that specified by the original, unmutated codon.

An **intergenic suppressor mutation**, in contrast, occurs in a gene other than the one bearing the original mutation. These suppressors sometimes work by changing the way that the mRNA is translated.

Below is a summary of these mutations

![image](https://user-images.githubusercontent.com/98819046/153392259-24a23790-3cd5-49ad-999a-e2d8d93cb21b.png)













